src/ontology/uberon-edit.obo

```diff
@@ -224701,6 +224701,16 @@ intersection_of: connects UBERON:7500034 ! distal-most point of trochlea of hume
 intersection_of: connects UBERON:7500048 ! proximal-most point of ventral tubercle of humerus
 created_by: https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500062
+name: tibial tuberosity
+def: "The tibial tuberosity is on the anterior side of a tibia and is where a patella ligament attaches." []
```

Comment:
I thought Uberon required a dbxref for the def, but maybe not, since this passed the QC checks. But if you have a reference, that would be nice to include.